tests/test_solution_variables.py

```diff
@@ -278,116 +199,6 @@ def test_svars(new_solver_session):
 
     fluid_temp = sv_p_wall_fluid["elbow-fluid"]
     assert fluid_temp.size == 17822
-    assert str(fluid_temp.dtype) == "float64"
-
-    wall_press_array = svar_data.create_empty_array("SV_P", "wall-elbow", "mixture")
-    fluid_press_array = svar_data.create_empty_array("SV_P", "elbow-fluid", "mixture")
-    wall_press_array[:] = 500
-    fluid_press_array[:] = 600
-    zone_names_to_svar_data = {
-        "wall-elbow": wall_press_array,
-        "elbow-fluid": fluid_press_array,
-    }
-    svar_data.set_svar_data(
-        svar_name="SV_P",
-        zone_names_to_svar_data=zone_names_to_svar_data,
-        domain_name="mixture",
-    )
-
-    updated_sv_p_data = svar_data.get_svar_data(
-        svar_name="SV_P",
-        zone_names=["elbow-fluid", "wall-elbow"],
-        domain_name="mixture",
-    )
-
-    assert updated_sv_p_data.domain == "mixture"
-    assert updated_sv_p_data.zones == ["wall-elbow", "elbow-fluid"]
-
-    assert updated_sv_p_data["elbow-fluid"].size == 17822
-    assert str(updated_sv_p_data["elbow-fluid"].dtype) == "float64"
-
-    assert updated_sv_p_data["wall-elbow"][0] == 500.0
-    assert updated_sv_p_data["elbow-fluid"][-1] == 600.0
-
-
-@pytest.mark.fluent_version(">=23.2")
-def test_svars_single_precision(new_solver_session_single_precision):
```

Comment:
At first sight, it seems that this `test_svars_single_precision` and `test_solution_variables_single_precision` were basically duplicates of each other, no? Wonder why both tests were running

`test_svars` and `test_solution_variables` too